experimental/adaptive-dialog/docs/anatomy-and-runtime-behavior.md

```diff
@@ -0,0 +1,82 @@
+# Adaptive dialog: Anatomy and runtime behavior
+
+## Anatomy: Adaptive dialog
+
+***Adaptive dialogs*** at the core comprise of 4 main concepts - 
+- [Recognizers](#Recognizers)
+- [Rules](#Rules)
+- [Steps](#Steps)
+- [Inputs](#Inputs)
+
+<p align="center">
+    <img alt="Adaptive_dialog_anatomy" src="./Assets/adaptive-dialog-anatomy.png" style="max-width:700px;" />
+</p>
+
+### Recognizers
+_Recognizers_ help understand and extract meaningful pieces of information from user's input. All recognizers emit events - of specific interest is the 'recognizedIntent' event that fires when the recognizer picks up an intent (or extracts entities) from given user utterance. See [here][1] to learn more about supported recognizers and their usage.
+
+### Rules
+_Rules_ enable you to catch and respond to events. The broadest rule is the EventRule that allows you to catch and attach a set of steps to execute when a specific event is emitted by any sub-system. Adaptive dialogs supports couple of other specialized rules to wrap common events that your bot would handle. See [here][2] to learn more about supported Rules and their usage.
+
+### Steps
+_Steps_ help put together the flow of conversation when a specific event is captured via a Rule. **_Note:_** unlike Waterfall dialog where each step is a function, each step in an Adaptive dialog is in itself a dialog. This enables adaptive dialogs by design to 
```

Comment:
I am not sure what is the value in "This enables adaptive dialogs by design to "